packages/mobile/README.md

```diff
@@ -67,13 +72,20 @@ for logging in with Google credentials.
 To test your GCP access, try running `yarn keys:decrypt` from the wallet repo root. You should see something like this: `Encrypted files decrypted`.
 (You will not need to run this command on an ongoing basis, since it is done automatically as part of the `postinstall` script.)
 
+#### For External contributors
+
+External contributors don't need to decrypt repository secrets and can successfully build and run the mobile application with the following differences:
+
+- the default branding will be used (some images/icons will appear in pink or will be missing)
+- Firebase related features needs to be disabled. You can do this by setting `FIREBASE_ENABLED=true` in the `packages/mobile/.env.*` files.
```

Comment:
Should this say `FIREBASE_ENABLED=false`?

Reply:
yep thanks!